cli/tests/testdata/worker_message_before_close.js

```diff
@@ -5,11 +5,12 @@ for (let i = 0; i < 4; i++) {
   );
 
   worker.addEventListener("message", (message) => {
-    // Only print responses after all reception logs.
+    // Only print responses in order after all reception logs.
     setTimeout(() => {
       console.log("response from worker %d received", message.data);
-    }, 500);
+    }, 500 * (i + 1));
```

Comment:
Rather than this setTimeout, wouldn't it suffice to just increase a counter and log something when all are received? The setTimeout seems unnecessary.